### PR TITLE
[candi] Add support of ROSA server

### PR DESCRIPTION
**Pull request by borg-z**
## Description
Added support for Rosa Server (Chrome and Cobalt) 

## Why do we need it, and what problem does it solve?
This update ensures compatibility with the latest version of Rosa Server

## Why do we need it in the patch release (if we do)?
Not necessarily.

## What is the expected result?
The system should now correctly detect and install necessary packages for Rosa Server

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: feature
summary: Added support for Rosa Server
impact: Support for Rosa Server ensures compatibility with the latest OS version, providing updated packages and configurations.
impact_level: default
```

### CODE REVIEW

File: ee/be/candi/bashible/detect_bundle.sh [hunk 1]
```diff
@@ -28,11 +28,17 @@ case "$ID" in
     name_is_not_supported
   ;;
   redos)
-    case "$VERSION_ID" in 7|7.*|8|8.*)
+    case "$VERSION_ID" in 7|7.*)
```
Comment (RomanenkoDenys) [inline ee/be/candi/bashible/detect_bundle.sh:31]: Why you remove Redos 8?

File: ee/be/modules/600-flant-integration/images/flant-pricing/hooks/prometheus_metrics.sh [hunk 2]
```diff
@@ -328,7 +328,8 @@ function node_os_image_metrics() {
                 "(?:(ubuntu).*(18.04|20.04|22.04))",
                 "(?:(rocky).*(8|9))",
                 "(?:(almalinux).*(7|8))",
-                "(?:(redos).*(7))",
```
Comment (RomanenkoDenys) [inline ee/be/modules/600-flant-integration/images/flant-pricing/hooks/prometheus_metrics.sh:331]: @asviel please give attention

File: ee/be/candi/bashible/bundles/rels/all/002_install_ca_certificates.sh.tpl [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Flant JSC
```
Comment (RomanenkoDenys) [inline ee/be/candi/bashible/bundles/rels/all/002_install_ca_certificates.sh.tpl:1]:
```suggestion
# Copyright 2024 Flant JSC
```
Please change in all files for Rosa bundle

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: ee/be/modules/600-flant-integration/images/flant-pricing/hooks/prometheus_metrics.sh (hunk 2 above)
Comment (asviel) [reply ee/be/modules/600-flant-integration/images/flant-pricing/hooks/prometheus_metrics.sh:331]: thx

File: ee/be/candi/bashible/detect_bundle.sh (hunk 1 above)
Comment (borg-z) [reply ee/be/candi/bashible/detect_bundle.sh]: resl <-> redos

**Review by RomanenkoDenys — APPROVED** [review]

File: ee/be/candi/bashible/bundles/rels/node-group/004_install_mandatory_packages_for_cloud_nodes.sh.tpl [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{{- if ne .nodeGroup.nodeType "Static" }}
+# Copyright 2024 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE.
+
+bb-yum-install cloud-utils-growpart
```
[the remark this answers is not in the document]
Comment (sprait) [reply ee/be/candi/bashible/bundles/rels/node-group/004_install_mandatory_packages_for_cloud_nodes.sh.tpl:5]: we can try it. I see that growpart it is shell script, which used binaries sfdisk/sgdisk

[the remark this answers is not in the document]
Comment (RomanenkoDenys) [reply ee/be/candi/bashible/bundles/rels/node-group/004_install_mandatory_packages_for_cloud_nodes.sh.tpl:5]: we should do not install unneeded binaries, so we need solution to do growpart

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/gcp'...

**Review by RomanenkoDenys — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Pull request merged**